src/routes/landing/components/Heading.js

```diff
@@ -16,11 +26,35 @@ const Wrapper: StyledComponent<{| children: React.Node |}, {}, *> = styled.View`
   align-items: center;
 `
 
-class Heading extends React.Component<{||}> {
+class Heading extends React.Component<PropsType, StateType> {
+  constructor (props: PropsType) {
+    super(props)
+    this.state = { clickCount: 0 }
+  }
+
+  onImagePress = async () => {
+    const prevClickCount = this.state.clickCount
+    if (prevClickCount + 1 >= API_URL_OVERRIDE_MIN_CLICKS) {
+      const appSettings = new AppSettings()
+      const apiUrlOverride = await appSettings.loadApiUrlOverride()
+      const newApiUrl = apiUrlOverride === testBaseUrl ? liveBaseUrl : testBaseUrl
+      await appSettings.setApiUrlOverride(newApiUrl)
+      this.setState({ clickCount: 0 })
+      this.showApiUrlToast(newApiUrl)
+      this.props.clearResourcesAndCache()
+    } else {
+      this.setState(previousState => ({ clickCount: previousState.clickCount + 1 }))
+    }
+  }
+
+  showApiUrlToast = (apiUrl: string) => ToastAndroid.show(`Switched to new API-Url: ${apiUrl}`, ToastAndroid.LONG)
```

Comment:
What happens on iOS here? It should at least not crash.
Alternatively you could use an alert and also tell the user how to return to the live cms.

Reply:
I'm now using `react-native-simple-toast` which should also display something toast-like on IOS. Didn't test it though due to a lacking IOS device.